### PR TITLE
python3Packages.qiskit: 0.20.0 -> 0.23.1, update dependencies & subpackages

### PR DESCRIPTION
**Pull request by drewrisinger**
###### Motivation for this change
* Update to latest version of qiskit.
* Update qiskit dependencies: ``fastjsonschema, pproxy``.
* Turn off optional visualization packages by default: reduce runtime closure & improve import speed (https://github.com/Qiskit/qiskit-terra/issues/5100)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/qiskit-ibmq-provider/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/qiskit-ibmq-provider/default.nix:59]: otherwise you will have an shape `[[derivation]]`
```suggestion
  ] ++ lib.optionals withVisualization visualizationPackages;
```

File: pkgs/development/python-modules/qiskit-ibmq-provider/default.nix
Comment (jonringer) [inline pkgs/development/python-modules/qiskit-ibmq-provider/default.nix:68]: this will be added twice to the check phase if withVisualization is passed
```suggestion
  ] ++ lib.optionals (!withVisualization) visualizationPackages;
```

Comment (drewrisinger) [reply pkgs/development/python-modules/qiskit-ibmq-provider/default.nix:68]: Ok, I assumed there was some sort of unique filter when adding packages, but I have no objection.

**Comment by drewrisinger** [comment]
Converted to draft pending update of qiskit to 0.23.1 later this week.

**Comment by jonringer** [comment]
getting qiskit failures

```
  ERROR: Could not find a version that satisfies the requirement qiskit-aqua==0.8.0 (from qiskit==0.23.0) (from versions: none)
  ERROR: No matching distribution found for qiskit-aqua==0.8.0 (from qiskit==0.23.0)
cannot build derivation '/nix/store/nxpm2b6z71kp0vakgv5xd6a2maw7v7xa-env.drv': 2 dependencies couldn't be built
[32 built (2 failed), 338 copied (1314.0 MiB), 221.7 MiB DL]
error: build of '/nix/store/nxpm2b6z71kp0vakgv5xd6a2maw7v7xa-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/100007
2 packages marked as broken and skipped:
python37Packages.qasm2image python38Packages.qasm2image

2 packages failed to build:
python37Packages.qiskit python38Packages.qiskit

30 packages built:
google-music-scripts home-assistant python27Packages.docplex python27Packages.pylatexenc python37Packages.docplex python37Packages.fastjsonschema python37Packages.google-music python37Packages.httpcore python37Packages.httpx python37Packages.pproxy python37Packages.pylatexenc python37Packages.qiskit-aer python37Packages.qiskit-aqua python37Packages.qiskit-ibmq-provider python37Packages.qiskit-ignis python37Packages.qiskit-terra python37Packages.retworkx python38Packages.docplex python38Packages.fastjsonschema python38Packages.google-music python38Packages.httpcore python38Packages.httpx python38Packages.pproxy python38Packages.pylatexenc python38Packages.qiskit-aer python38Packages.qiskit-aqua python38Packages.qiskit-ibmq-provider python38Packages.qiskit-ignis python38Packages.qiskit-terra python38Packages.retworkx
```

**Comment by drewrisinger** [comment]
@jonringer it's still marked as draft for that reason. Those failures are b/c I updated the version of ``qiskit-aqua`` in advance of them releasing a new version of the metapackage. The metapackage checks that the subpackages have the correct version (which is now greater than expected), hence the failure. I'm waiting for the new metapackage release probably this week.

**Comment by drewrisinger** [comment]
@jonringer I built all of these packages locally, so it should be good for review. This can also test for #102724, the only thing missing before merge is removing the ``testing: timeout long tests`` commit (maybe, unless you'd prefer to have it in there for regular builds, but that could be flaky depending on server load) pending your review.

**Comment by SuperSandro2000** [comment]
Fails to build for me:

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing setuptools-check-hook
Using setuptoolsCheckPhase
Sourcing pytest-check-hook
Using pytestCheckPhase
Removing setuptoolsCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/89haipgdpixd2pm6pcal9ayhbvvn1l9n-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tools/unicode.xml.LICENSE
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
Executing setuptoolsBuildPhase
running bdist_wheel
running build
running build_py
creating build
creating build/lib
creating build/lib/pylatexenc
copying pylatexenc/version.py -> build/lib/pylatexenc
copying pylatexenc/_util.py -> build/lib/pylatexenc
copying pylatexenc/__init__.py -> build/lib/pylatexenc
creating build/lib/pylatexenc/macrospec
copying pylatexenc/macrospec/_argparsers.py -> build/lib/pylatexenc/macrospec
copying pylatexenc/macrospec/__init__.py -> build/lib/pylatexenc/macrospec
creating build/lib/pylatexenc/latexwalker
copying pylatexenc/latexwalker/_defaultspecs.py -> build/lib/pylatexenc/latexwalker
copying pylatexenc/latexwalker/__main__.py -> build/lib/pylatexenc/latexwalker
copying pylatexenc/latexwalker/__init__.py -> build/lib/pylatexenc/latexwalker
creating build/lib/pylatexenc/latexencode
copying pylatexenc/latexencode/__init__.py -> build/lib/pylatexenc/latexencode
copying pylatexenc/latexencode/_uni2latexmap_xml.py -> build/lib/pylatexenc/latexencode
copying pylatexenc/latexencode/_uni2latexmap.py -> build/lib/pylatexenc/latexencode
copying pylatexenc/latexencode/__main__.py -> build/lib/pylatexenc/latexencode
creating build/lib/pylatexenc/latex2text
copying pylatexenc/latex2text/_defaultspecs.py -> build/lib/pylatexenc/latex2text
copying pylatexenc/latex2text/__init__.py -> build/lib/pylatexenc/latex2text
copying pylatexenc/latex2text/__main__.py -> build/lib/pylatexenc/latex2text
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/pylatexenc
creating build/bdist.linux-x86_64/wheel/pylatexenc/latex2text
copying build/lib/pylatexenc/latex2text/__main__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latex2text
copying build/lib/pylatexenc/latex2text/__init__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latex2text
copying build/lib/pylatexenc/latex2text/_defaultspecs.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latex2text
creating build/bdist.linux-x86_64/wheel/pylatexenc/latexencode
copying build/lib/pylatexenc/latexencode/__main__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexencode
copying build/lib/pylatexenc/latexencode/_uni2latexmap.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexencode
copying build/lib/pylatexenc/latexencode/_uni2latexmap_xml.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexencode
copying build/lib/pylatexenc/latexencode/__init__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexencode
creating build/bdist.linux-x86_64/wheel/pylatexenc/latexwalker
copying build/lib/pylatexenc/latexwalker/__init__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexwalker
copying build/lib/pylatexenc/latexwalker/__main__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexwalker
copying build/lib/pylatexenc/latexwalker/_defaultspecs.py -> build/bdist.linux-x86_64/wheel/pylatexenc/latexwalker
creating build/bdist.linux-x86_64/wheel/pylatexenc/macrospec
copying build/lib/pylatexenc/macrospec/__init__.py -> build/bdist.linux-x86_64/wheel/pylatexenc/macrospec
copying build/lib/pylatexenc/macrospec/_argparsers.py -> build/bdist.linux-x86_64/wheel/pylatexenc/macrospec
copying build/lib/pylatexenc/__init__.py -> build/bdist.linux-x86_64/wheel/pylatexenc
copying build/lib/pylatexenc/_util.py -> build/bdist.linux-x86_64/wheel/pylatexenc
copying build/lib/pylatexenc/version.py -> build/bdist.linux-x86_64/wheel/pylatexenc
running install_egg_info
running egg_info
creating pylatexenc.egg-info
writing pylatexenc.egg-info/PKG-INFO
writing top-level names to pylatexenc.egg-info/top_level.txt
writing dependency_links to pylatexenc.egg-info/dependency_links.txt
writing entry points to pylatexenc.egg-info/entry_points.txt
writing manifest file 'pylatexenc.egg-info/SOURCES.txt'
reading manifest file 'pylatexenc.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'pylatexenc.egg-info/SOURCES.txt'
Copying pylatexenc.egg-info to build/bdist.linux-x86_64/wheel/pylatexenc-2.8-py2.7.egg-info
running install_scripts
adding license file "LICENSE.txt" (matched pattern "LICEN[CS]E*")
creating build/bdist.linux-x86_64/wheel/pylatexenc-2.8.dist-info/WHEEL
creating 'dist/pylatexenc-2.8-py2-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'pylatexenc/__init__.py'
adding 'pylatexenc/_util.py'
adding 'pylatexenc/version.py'
adding 'pylatexenc/latex2text/__init__.py'
adding 'pylatexenc/latex2text/__main__.py'
adding 'pylatexenc/latex2text/_defaultspecs.py'
adding 'pylatexenc/latexencode/__init__.py'
adding 'pylatexenc/latexencode/__main__.py'
adding 'pylatexenc/latexencode/_uni2latexmap.py'
adding 'pylatexenc/latexencode/_uni2latexmap_xml.py'
adding 'pylatexenc/latexwalker/__init__.py'
adding 'pylatexenc/latexwalker/__main__.py'
adding 'pylatexenc/latexwalker/_defaultspecs.py'
adding 'pylatexenc/macrospec/__init__.py'
adding 'pylatexenc/macrospec/_argparsers.py'
adding 'pylatexenc-2.8.dist-info/LICENSE.txt'
adding 'pylatexenc-2.8.dist-info/METADATA'
adding 'pylatexenc-2.8.dist-info/WHEEL'
adding 'pylatexenc-2.8.dist-info/entry_points.txt'
adding 'pylatexenc-2.8.dist-info/top_level.txt'
adding 'pylatexenc-2.8.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Finished executing setuptoolsBuildPhase
@nix { "action": "setPhase", "phase": "installPhase" }
installing
Executing pipInstallPhase
/build/source/dist /build/source
[33mDEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.[0m
[33mDEPRECATION: The -b/--build/--build-dir/--build-directory option is deprecated. pip 20.3 will remove support for this functionality. A possible replacement is use the TMPDIR/TEMP/TMP environment variable, possibly combined with --no-clean. You can find discussion regarding this at https://github.com/pypa/pip/issues/8333.[0m
Processing ./pylatexenc-2.8-py2-none-any.whl
Installing collected packages: pylatexenc
[33m  WARNING: The scripts latex2text, latexencode and latexwalker are installed in '/nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.[0m
Successfully installed pylatexenc-2.8
/build/source
Finished executing pipInstallPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8
strip is /nix/store/p3kn26g5nhmx6spn37ar3mn0xjbzks3g-binutils-2.31.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/lib  /nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/bin 
patching script interpreter paths in /nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8
checking for references to /build/ in /nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8...
Rewriting #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/bin/python2.7 to #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18
wrapping `/nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/bin/latex2text'...
Rewriting #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/bin/python2.7 to #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18
wrapping `/nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/bin/latexwalker'...
Rewriting #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/bin/python2.7 to #!/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18
wrapping `/nix/store/kdrvwcrykdy5inrlk9dbr6c6d97f34jk-python2.7-pylatexenc-2.8/bin/latexencode'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
no Makefile or custom installCheckPhase, doing nothing
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: pylatexenc
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
pytestCheckPhase
Executing pytestCheckPhase
Traceback (most recent call last):
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/pytest.py", line 8, in <module>
    from _pytest.assertion import register_assert_rewrite
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/_pytest/assertion/__init__.py", line 13, in <module>
    from _pytest.assertion import rewrite
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/_pytest/assertion/rewrite.py", line 24, in <module>
    from _pytest.assertion import util
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/_pytest/assertion/util.py", line 11, in <module>
    import _pytest._code
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/_pytest/_code/__init__.py", line 7, in <module>
    from .code import Code  # noqa
  File "/nix/store/9pljl7zv2aqg5685lnqznlfq43ri8xzc-python2.7-pytest-4.6.11/lib/python2.7/site-packages/_pytest/_code/code.py", line 15, in <module>
    import pluggy
  File "/nix/store/hkryh6l33n2vkdydv8fmyll4znv6rw2f-python2.7-pluggy-0.13.1/lib/python2.7/site-packages/pluggy/__init__.py", line 16, in <module>
    from .manager import PluginManager, PluginValidationError
  File "/nix/store/hkryh6l33n2vkdydv8fmyll4znv6rw2f-python2.7-pluggy-0.13.1/lib/python2.7/site-packages/pluggy/manager.py", line 11, in <module>
    import importlib_metadata
  File "/nix/store/vgaqq0mq904m25sy507lp43lbfkwfzzg-python2.7-importlib-metadata-1.7.0/lib/python2.7/site-packages/importlib_metadata/__init__.py", line 16, in <module>
    from ._compat import (
  File "/nix/store/vgaqq0mq904m25sy507lp43lbfkwfzzg-python2.7-importlib-metadata-1.7.0/lib/python2.7/site-packages/importlib_metadata/_compat.py", line 36, in <module>
    import pathlib2 as pathlib
  File "/nix/store/0in6k74biz5cj382hng7i5327l40fmhx-python2.7-pathlib2-2.3.5/lib/python2.7/site-packages/pathlib2/__init__.py", line 52, in <module>
    from scandir import scandir as os_scandir
  File "/nix/store/jga45w64gzlnrag6ad5s5pnjab3r28w6-python2.7-scandir-1.10.0/lib/python2.7/site-packages/scandir.py", line 452, in <module>
    libc = ctypes.CDLL(ctypes.util.find_library('c'), use_errno=True)
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/ctypes/util.py", line 241, in find_library
    return _findSoname_ldconfig(name) or _get_soname(_findLib_gcc(name))
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/ctypes/util.py", line 103, in _findLib_gcc
    stdout=subprocess.PIPE)
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/nix/store/vfd4wghhzz4afzxgwynsnbj9xzpkib4z-python-2.7.18/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 100007` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python37Packages.qasm2image</li>
    <li>python38Packages.qasm2image</li>
  </ul>
</details>
<details>
  <summary>10 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python27Packages.pylatexenc</li>
    <li>python37Packages.qiskit</li>
    <li>python37Packages.qiskit-aer</li>
    <li>python37Packages.qiskit-aqua</li>
    <li>python37Packages.qiskit-ignis</li>
    <li>python38Packages.qiskit</li>
    <li>python38Packages.qiskit-aer</li>
    <li>python38Packages.qiskit-aqua</li>
    <li>python38Packages.qiskit-ignis</li>
  </ul>
</details>
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>google-music-scripts</li>
    <li>python27Packages.docplex</li>
    <li>python37Packages.docplex</li>
    <li>python37Packages.fastjsonschema</li>
    <li>python37Packages.google-music</li>
    <li>python37Packages.httpcore</li>
    <li>python37Packages.httpx</li>
    <li>python37Packages.pproxy</li>
    <li>python37Packages.pylatexenc</li>
    <li>python37Packages.qiskit-ibmq-provider</li>
    <li>python37Packages.qiskit-terra</li>
    <li>python37Packages.retworkx</li>
    <li>python38Packages.docplex</li>
    <li>python38Packages.fastjsonschema</li>
    <li>python38Packages.google-music</li>
    <li>python38Packages.httpcore</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.pproxy</li>
    <li>python38Packages.pylatexenc</li>
    <li>python38Packages.qiskit-ibmq-provider</li>
    <li>python38Packages.qiskit-terra</li>
    <li>python38Packages.retworkx</li>
  </ul>
</details>

**Comment by drewrisinger** [comment]
@SuperSandro2000 I can't replicate the errors. Here's a fresh rebuild from today (``python27Packages.docplex`` fails due to security issues with ``python27Packages.cryptography``).

```
https://github.com/NixOS/nixpkgs/pull/100007
1 package marked as broken and skipped:
python27Packages.docplex

33 packages built:
google-music-scripts home-assistant python27Packages.pylatexenc python37Packages.docplex python37Packages.fastjsonschema python37Packages.google-music python37Packages.httpcore python37Packages.httpx python37Packages.pproxy python37Packages.pylatexenc python37Packages.qasm2image python37Packages.qiskit python37Packages.qiskit-aer python37Packages.qiskit-aqua python37Packages.qiskit-ibmq-provider python37Packages.qiskit-ignis python37Packages.qiskit-terra python37Packages.retworkx python38Packages.docplex python38Packages.fastjsonschema python38Packages.google-music python38Packages.httpcore python38Packages.httpx python38Packages.pproxy python38Packages.pylatexenc python38Packages.qasm2image python38Packages.qiskit python38Packages.qiskit-aer python38Packages.qiskit-aqua python38Packages.qiskit-ibmq-provider python38Packages.qiskit-ignis python38Packages.qiskit-terra python38Packages.retworkx

$ nix-shell /.../.cache/nixpkgs-review/pr-100007-1/shell.nix
```

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
https://github.com/NixOS/nixpkgs/pull/100007
3 packages marked as broken and skipped:
python27Packages.docplex python37Packages.qasm2image python38Packages.qasm2image

31 packages built:
google-music-scripts home-assistant python27Packages.pylatexenc python37Packages.docplex python37Packages.fastjsonschema python37Packages.google-music python37Packages.httpcore python37Packages.httpx python37Packages.pproxy python37Packages.pylatexenc python37Packages.qiskit python37Packages.qiskit-aer python37Packages.qiskit-aqua python37Packages.qiskit-ibmq-provider python37Packages.qiskit-ignis python37Packages.qiskit-terra python37Packages.retworkx python38Packages.docplex python38Packages.fastjsonschema python38Packages.google-music python38Packages.httpcore python38Packages.httpx python38Packages.pproxy python38Packages.pylatexenc python38Packages.qiskit python38Packages.qiskit-aer python38Packages.qiskit-aqua python38Packages.qiskit-ibmq-provider python38Packages.qiskit-ignis python38Packages.qiskit-terra python38Packages.retworkx
```

**Pull request merged**
